Library/Homebrew/extend/pathname.rb

```diff
@@ -82,6 +82,19 @@ class Pathname
 
   include DiskUsageExtension
 
+  CELLAR_PATH_REGEX = %r{(?<prefix>#{HOMEBREW_CELLAR}/?(?<name>[^/]+)?)/?(?<version>[^/]+)?(?<suffix>/.*)?}.freeze
+
+  # Converts an absolute Cellar path into the equivalent opt path.
+  # No-op if {Pathname} is not a descendant of HOMEBREW_CELLAR
+  # or if {Pathname} is a relative path.
+  sig { returns(Pathname) }
+  def to_opt
```

Comment:
Do you plan to use this in formulae? If so, I'd prefer not to have this `extend Pathname`. If not, can you make it `private` (and perhaps consider pulling it into `utils`)?

Reply:
No, there's no reason to use it in formulae, I think. Let me look at moving it/making it private.